apps/mudita-center-e2e/src/page-objects/modal.page.ts

```diff
@@ -2,26 +2,27 @@
  * Copyright (c) Mudita sp. z o.o. All rights reserved.
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
-import { ChainablePromiseElement } from "webdriverio"
+
 /**
  * main page object containing all methods, selectors and functionality
  * that is shared across all page objects
  */
+
 export default class ModalPage {
-  public get modalHeader(): ChainablePromiseElement<
-    Promise<WebdriverIO.Element>
-  > {
+  public static get modalHeader() {
     return $('[data-testid="modal-header"]')
   }
 
-  public get modalCloseButton(): ChainablePromiseElement<
-    Promise<WebdriverIO.Element>
-  > {
+  public static get modalCloseButton() {
     return this.modalHeader.$('[data-testid="icon-Close"]')
   }
 
-  async closeModalButtonClick() {
-    await this.modalCloseButton.waitForClickable()
-    await this.modalCloseButton.click()
+  public static get modalOverlay() {
+    return $("div.ReactModal__Overlay")
```

Comment:
Should we ask for data-testid here?

Reply:
Using the `.ReactModal__Overlay` selector is fine here since it refers to a specific component that isn’t modified by Styled Components, ensuring consistent selectors across builds. However, we should consider whether we need the entire `Modal.Page`, which currently applies to any modal, not just a specific one. This might require refactoring, which would be a good topic for a separate task.